protocols/virtualfund/virtualfund.go

```diff
@@ -132,6 +133,21 @@ type Objective struct {
 	b0 types.Funds // Initial balance for Bob
 }
 
+// onlyPayerHasFunds checks that the outcome only has funds allocated to the payer
+func onlyPayerHasFunds(s state.State) bool {
+	for _, e := range s.Outcome {
+		total := e.TotalAllocated()
+		for i, a := range e.Allocations {
+			if i == payments.PAYER_INDEX && a.Amount.Cmp(total) != 0 {
+				return false
+			} else if i != payments.PAYER_INDEX && a.Amount.Cmp(big.NewInt(0)) != 0 {
```

Comment:
Super nit but you don't need the `else` here, and I believe it is encouraged to omit it (saves some indentation).